pingcap/tidb
# ddl: scattering truncated tables without pre-split option

Signed-off-by: Shuaipeng Yu <jackysp@gmail.com>



### What problem does this PR solve?

Issue Number: close #xxx 

Problem Summary:
The current truncate table statement could not scatter regions for tables without pre-split option.

### What is changed and how it works?

Proposal: [xxx](url) 

What's Changed:
Make scattering regions for truncated tables without pre-split option.

How it Works:

### Check List 

Tests 

- Manual test (add detailed scripts or steps below)
  * start a cluster
  * truncate some tables which have no pre-split option
  * grep the log files of pd server for 'add operator', it will have 'transfer leader' and 'mv light peer' (will not see mv operator because misuse `ScatterRegions` api in `pdclient`, see #22788).

### Release note 

- Scattering truncated tables without pre-split option.
